build.properties.dist

```diff
@@ -148,6 +149,9 @@ phpcs.prepush.destination = ${project.basedir}/.git/hooks/pre-push
 # Development modules to enable.
 drupal.modules.dev = config_update field_ui menu_ui views_ui
 
+# UAT modules to enable.
+drupal.modules.uat = demo_content
+
```

Comment:
Good solution! Can you change the documentation so it's clear to people that do not know what UAT means? Maybe "Modules to enable on acceptance environments".